tests/test_trusted_metadata_set.py

```diff
@@ -471,6 +494,53 @@ def target_expired_modifier(target: Targets) -> None:
 
     # TODO test updating over initial metadata (new keys, newer timestamp, etc)
 
+    def test_load_from_simple_envelope(self) -> None:
+        """Basic unit test for ``_load_from_simple_envelope`` helper.
+
+        TODO: Test via trusted metadata set tests like for traditional metadata
```

Comment:
should make a ticket about this I suppose

Reply:
see #2565